.github/workflows/tests.yml

```diff
@@ -75,6 +75,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        with:
+          key: ddev-test-cache
+          path: ~/.ddev/testcache
```

Comment:
This looks great.

Reply:
Almost. This does not work correctly since the cache never gets updated with this key. It's important to include something like a look file here to get it working.